tests/robot-tests/tests/admin/bau/prerelease.robot

```diff
@@ -0,0 +1,129 @@
+    user checks summary list contains  Current status  Approved
+    user checks summary list contains  Scheduled release  ${PUBLISH_DATE_DAY} ${PUBLISH_DATE_MONTH_WORD} ${PUBLISH_DATE_YEAR}
+    user checks summary list contains  Next release expected  January 2001
+    user waits for release process status to be  Scheduled  90
+
+Navigate to Prerelease page
+    [Tags]  HappyPath
+    ${current_url}=  get location
+    ${release_url}=  remove substring from right of string  ${current_url}  /status
+    ${prerelease_url}=   evaluate  "${release_url}" + "/prerelease"
+    user goes to url   ${prerelease_url}
+
+Validate Prerelease page is displayed
+    [Tags]  HappyPath
+    user waits until page contains heading 1  Pre Release access is not yet available
+    user checks element count is x  css:[data-testid="breadcrumbs--list"] li     2
+    user checks element should contain  css:[data-testid="breadcrumbs--list"] li:nth-child(1)   Home
+    user checks element should contain  css:[data-testid="breadcrumbs--list"] li:nth-child(2)   Pre Release access
+
+Validate Prerelease page displays correct message
+    [Tags]  HappyPath   NotAgainstLocal
+    ${PREREL_DATE_DAY}=         get datetime  %d  1
+    ${PREREL_DATE_MONTH_WORD}=  get datetime  %B  1
+    ${PREREL_DATE_YEAR}=        get datetime  %Y  1
+    user checks page contains   Pre Release access will be available from ${PREREL_DATE_DAY} ${PREREL_DATE_MONTH_WORD} ${PREREL_DATE_YEAR} at 09:30 until ${PREREL_DATE_DAY} ${PREREL_DATE_MONTH_WORD} ${PREREL_DATE_YEAR} at 23:59.
+
+Navigate to Admin dashboard
+    [Tags]  HappyPath
+    user goes to url  %{ADMIN_URL}
+    user waits until page contains heading 1  Dashboard
+
+Go to View scheduled releases tab
+    [Tags]  HappyPath
+    user waits until page contains element  id:scheduled-releases-tab    60
+    user clicks element   id:scheduled-releases-tab
+    user waits until page contains element   xpath://*[@id="scheduled-releases-tab" and @aria-selected="true"]
+
+Invite a new user to prerelease for scheduled release
+    [Tags]  HappyPath
+    set test variable  ${details_selector}  xpath://*[@data-testid="releaseByStatusTab ${PUBLICATION_NAME}"]/details/summary//*[text()="Calendar Year 2000 (not Live)"]
+    user waits until page contains element  ${details_selector}
+    user waits until element is visible  ${details_selector}
+    user waits until element is enabled  ${details_selector}
+    user clicks element   ${details_selector}
+
+    ${details_dropdown}=   get webelement   xpath://*[@data-testid="releaseByStatusTab ${PUBLICATION_NAME}"]/details
+    user waits until parent contains element   ${details_dropdown}   xpath:.//*[text()="Manage pre release access"]
+    ${invite_input}=  get child element   ${details_dropdown}   xpath:.//label[text()="Invite a new user"]/../input
+
+    user waits until element is visible   ${invite_input}
+    user clicks element  ${invite_input}
+    user presses keys  mark@hiveit.co.uk
```

Comment:
Isn't this going to spam you with invites on dev?

Reply:
yes, but I don't know what other email to use. My email is already being used for a notifications test - but I have a folder all the emails automatically get moved to.

I'll create a ticket to address this.